pkg/migrations/op_common_test.go

```diff
@@ -40,6 +44,14 @@ func ExecuteTests(t *testing.T, tests TestCases, opts ...roll.Option) {
 	testSchema := testutils.TestSchema()
 
 	for _, tt := range tests {
+		if tt.minPgMajorVersion != 0 && os.Getenv("POSTGRES_VERSION") != "" {
+			pgMajorVersion := strings.Split(os.Getenv("POSTGRES_VERSION"), ".")[0]
+			version, _ := strconv.Atoi(pgMajorVersion)
```

Comment:
`POSTGRES_VERSION` is set to `latest` as part of the CI test matrix, ie not necessarily a Major.Minor string.

It's really a Postgres docker image tag, not strictly a postgres version number.

Maybe put this logic in a function and handle the case where it isn't Major.Minor release number.